### ISSUE TITLE
Bug or issue? Fatal error: Cannot use object of type WP_Error as array in

### ISSUE THREAD

**Issue by Tanckom** [flowdee/edd-software-license-manager#13]
Hello, so I've been previously using your tool, and it worked fine.
However I tested it now today again, and got this error when purchasing.
Any idea how to fix this?

`Fatal error: Cannot use object of type WP_Error as array in /users/cetagram/www/wordpress/wp-content/plugins/edd-software-license-manager/includes/purchase.php on line 105`

Link for purchase:
http://xs-sol.com/downloads/cetagram/

**Comment by btwatts** [comment]
Are you sure you included the value for number of license

How many sites can be activated trough[sic] a single license key?

If you leave this value blank, it generates errors.

Hope that helps!

On Thu, Jun 2, 2016 at 6:07 PM, Tanckom notifications@github.com wrote:

> Hello, so I've been previously using your tool, and it worked fine.
> However I tested it now today again, and got this error when purchasing.
> Any idea how to fix this?
> 
> Fatal error: Cannot use object of type WP_Error as array in
> /users/cetagram/www/wordpress/wp-content/plugins/edd-software-license-manager/includes/purchase.php
> on line 105
> 
> Link for purchase:
> http://xs-sol.com/downloads/cetagram/
> 
> The price is 0€ because i want to share it for free, however it needs to
> be licensed.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/flowdee/edd-software-license-manager/issues/13, or mute
> the thread
> https://github.com/notifications/unsubscribe/AHbeNiHPkX3EgeIpV0RgtCDTzKsvj2c4ks5qH2IggaJpZM4ItEOf
> .

## 

www.Byron.com             www.LifeShiftAcademy.com
www.ByronWatts.com    When you're ready to be your very best!
                                       Because _YOU Matter!_

**Comment by Tanckom** [comment]
Yes, i did that, and there can be two sites activated through a single license key.

It was all working fine several months ago, however when looking at it yesterday, i recieve this error...
